earth_enterprise/rpms/build.gradle

```diff
@@ -402,16 +402,84 @@ if (project.hasProperty('buildOpenGee')) {
     openGeeCommonRpm.dependsOn stageOpenGeeInstall
 }
 
-
-task openGeeCommonDeb (type: Deb) {
+task openGeeCommonDeb (type: GeeDeb, dependsOn: openGeeSharedFiles) {
     packageName = 'opengee-common'
+    release = "1.${rpmPlatformString}"
+    version = openGeeVersion
+    user = 'root'
+    permissionGroup = 'root'
+    packageGroup = 'Application/Productivity'
+    summary = 'Third-party libraries bundled with Open GEE'
+    // The package build failed when packageDescription was copied here from openGeeCommonRpm, so it is omitted for now
+    license =
```

Comment:
Can you please share values between RPM and Deb packages?  You could access the RPM task's properties, e.g. `openGeeCommonRpm.license`, or, if you prefer, you can make a class to hold the various properties for a package, so it can be shared between RPM and Deb packages.